CommunityFridgeMapApi/dependencies/python/db.py

```diff
@@ -198,11 +198,47 @@ def __init__(self, db_client: 'botocore.client.DynamoDB'):
         super().__init__(db_client=db_client)
 
 class FridgeHistory(DB_Item):
-    
+
     def __init__(self, db_client: 'botocore.client.DynamoDB'):
         super().__init__(db_client=db_client)
 
-class Tag(DB_Item):
 
-    def __init__(self, db_client: 'botocore.client.DynamoDB'):
+class Tag(DB_Item):
+    TABLE_NAME = "tag"
+    def __init__(self, db_client: 'botocore.client.DynamoDB', tag_name:str= None):
         super().__init__(db_client=db_client)
+
+        self.db_client = db_client
+        self.tag_name = tag_name
+
+    def set_tag(self, tag_name):
+        #Tag_name is alphanumeric, can include hyphen and underscore, with no spaces and all lower cased
+        tag_name = tag_name.lower().replace(" ", "")
```

Comment:
The thought process for me for this function is the tag_name may include spaces, capital letters, or other unacceptable characters; so, this function "sets" it in the appropriate form by removing the spaces, making it lower case, if needed, and checking that each character is acceptable. The latest commit does have the type hint for string but not a bool.  Regarding a crash, the test file does test a tag_name  with a None value and the test passes. Do you think additional tests are needed?

Reply:
1. Should we rename it to `format_tag()` then? Since we're just formatting something that was already set
2. I'm not talking about the tests. At runtime, `tag_name.lower()` will crash if `tag_name` is `None`. So either we need to check if `tag_name` is non-null, or handle the `None` value as well